Emby.Server.Implementations/LiveTv/TunerHosts/SharedHttpStream.cs

```diff
@@ -55,7 +55,8 @@ public override async Task Open(CancellationToken openCancellationToken)
             var typeName = GetType().Name;
             Logger.LogInformation("Opening " + typeName + " Live stream from {0}", url);
 
-            using var response = await _httpClientFactory.CreateClient(NamedClient.Default)
+            // Response stream is disposed manually.
+            var response = await _httpClientFactory.CreateClient(NamedClient.Default)
```

Comment:
The stream must be disposed in the thread that is reading it instead of the thread that started it. The reading thread disposes the stream manually here https://github.com/jellyfin/jellyfin/blob/2c6b31fc7bf45808ca041e9e76046495b7d351d6/Emby.Server.Implementations/LiveTv/TunerHosts/SharedHttpStream.cs#L131